Library/ViewModels/CommentsViewModel.swift

```diff
@@ -25,8 +25,11 @@ public protocol CommentsViewModelInputs {
 }
 
 public protocol CommentsViewModelOutputs {
-  /// Emits a CommentComposerViewData object that determines the avatarURL and whether the user is a backer.
-  var configureCommentComposerViewWithData: Signal<CommentComposerViewData, Never> { get }
+  /// Emits data to configure comment composer view.
+  var commentComposerData: Signal<CommentComposerViewData, Never> { get }
```

Comment:
Rename: `configureCommentComposerViewWithData`. This was its name before, not sure why the rename.